core/templates/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -203,6 +205,9 @@ angular.module('oppia').factory('ExplorationEngineService', [
           exploration.initStateName, newParams);
       }
 
+      StateClassifierMappingService.initializeClassifierDataForState(
```

Comment:
Can you put the classifier initialization and usage stuff behind a flag in constants.ts (ENABLE_ML_CLASSIFIERS or similar)? I think we'll want to go through a testing phase before we start using it fully in all the lessons.

Reply:
Done

Reply:
Where was this done? I can't find the flag in the changes.

Reply:
I have added the condition check as part of function call rather than covering the function call inside the condition.